docs/book/src/networking-introduction.md

```diff
@@ -0,0 +1,98 @@
+# Networking Introduction
+
+In this chapter, you will find a very short introduction to various networking concepts. 
+These concepts are important to understanding when to use QUIC.
+
+## 1. TCP/IP and UDP Comparison
+
+Lets define some transport guarantees and their properties:
+
+| Transport Guarantees | Packet Loss [(1)][1]  | Packet Duplication [(2)][2] | Packet Order | Packet Delivery |
+| :-------------:   | :-------------: | :-------------: | :-------------: | :-------------:
+|   **Unreliable**  |       Any       |      Yes        |     No          |   No
+|   **Reliable**    |       No        |      No         |     Ordered     |   Yes
+
+Now, lets compare TCP, UDP, and QUIC.
+
+| Feature |  TCP  | UDP | QUIC
+| :-------------: | :-------------: | :-------------: | :-------------: |
+|  [Connection-Oriented][6]           |       Yes         | No                       | Yes
+|  Transport Guarantees              | Reliable Ordered  | Unreliable               | Reliable Ordered and Unreliable 
+|  Packet Transfer                    | [Stream-based][4] | Message based            | Message based and/or Stream based
+|  Automatic [fragmentation][8]       | Yes               | Yes, ip-fragmentation    | Yes
+|  Header Size                        |  20 bytes         | 8 bytes                  |  16 bytes  
+|  [Control Flow, Congestion Avoidance/Control][5] | Yes  | No                       |  Yes, and user controlled                                          
+|  Based On | [IP][3]                | [IP][3]            |  UDP
+
+Unreliability gives great uncertainty with a lot of freedom, while reliability gives great certainty with costs in speed and freedom.
+That is why some protocols such as QUIC, SCTP are built on UDP instead of TCP. 
+UDP gives the end-user more control over the transmission than TCP can do. 
+While QUIC is build on top of UDP it does provides the same and even more features than TCP.
+
+## 2. Issues with TCP 
+
+While TCP has been around for long it does have some issues that QUIC tries to solve.
+
+### Head-of-line Blocking
+
+One of the biggest issues with TCP is that of Head-of-line blocking. 
+It is a convenient feature because it ensures that all packages are sent and arrive in order. 
+However, in cases of high throughput (multiplayer game networking) and big load in a short time (web page load), this can be catastrophic to your application performance.
```

Comment:
```suggestion
However, in cases of high throughput (multiplayer game networking) and big load in a short time (web page load), this can severely impact latency.
```